packages/compass-e2e-tests/helpers/telemetry.ts

```diff
@@ -32,25 +45,21 @@ async function startTelemetryServer() {
     delete process.env.HADRON_METRICS_SEGMENT_HOST_OVERRIDE;
   }
 
-  function events() {
+  function events(): Array<any> {
```

Comment:
```suggestion
  function events(): TelemetryEvent[] {
```

Reply:
This is actually not right. it is request.body.batch which is something like:

```
{
      userId: '90b7a520-82fd-475e-8258-a0d8b2d451d0',
      traits: [Object],
      type: 'identify',
      context: [Object],
      _metadata: [Object],
      timestamp: '2022-01-27T09:51:24.591Z',
      messageId: 'node-14885ba35ccc8e962b88fb092d3e75f4-c5262f1f-9bfd-4c0c-9be4-74da65dbaaf2'
    }
```

Reply:
Or I'm confused..